src/node_os.cc

```diff
@@ -247,6 +247,32 @@ static void GetInterfaceAddresses(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(Array::New(isolate, result.data(), result.size()));
 }
 
+static void GetOnLineStatus(const FunctionCallbackInfo<Value>& args) {
```

Comment:
Technically you can add V8 Fast API to this function and improve the performance by roughly 30%.

Reply:
I have no clue how to apply v8 fast api. Can you make a code suggestion, which i csn apply and i can learn from, please?

But on the other hand,  i think libuv is the bottleneck.

Reply:
We have a documentation explaining this: https://github.com/nodejs/node/blob/main/doc/contributing/adding-v8-fast-api.md

Reply:
not feasable